css-canvas-medical/styles.css

```diff
@@ -0,0 +1,244 @@
+.header {
+  display: flex;
+  margin-top: 25px;
+  margin-bottom: 10px;
+  padding-left: 25px;
+}
+
+.head-link,
+.contact-button {
+  font-size: 12px;
+  justify-self: center;
+  margin-left: 12px;
+  margin-right: 12px;
+  margin-top: 10px;
+}
+
+.contact-button {
+  background-color: rgb(29, 50, 95);
+  color: white;
+  border: 1px solid rgb(29, 50, 95);
+  border-radius: 3px;
+  padding: 7px 10px;
+  position: relative;
+  bottom: 6px;
+}
+
+.logo {
+  width: 150px;
+  margin-right: 240px;
+}
+
+p {
+  font-size: 10px;
+}
+
+p,
+button,
+.links,
+.foot-links,
+h6,
+.head-link {
+  font-family: 'Avenir', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.hero {
+  width: 100%;
+  height: 345px;
+  opacity: 0.9;
+  margin-left: 45px;
+  margin-right: 45px;
+}
+
+.get-started {
+  display: flex;
+  position: relative;
+  margin-bottom: 50px;
+  padding-right: 20px;
+  padding-left: 20px;
+}
+
+.g-s-button {
+  position: absolute;
```

Comment:
I think you can find a way with something like flexbox to do this without `absolute` positioning

Reply:
I am totally wrong about this, this is fine to have `absolute` 😂